tests/yam/agama/import_agama_profile.pm

```diff
@@ -18,6 +22,8 @@ sub run {
 
     select_console 'root-console';
     assert_script_run("agama profile import $profile_url", timeout => 300);
+    # start headless install
+    background_script_run('agama install > /dev/null 2>&1') if (get_var('EXTRABOOTPARAMS', '') =~ /systemd.unit=multi-user.target/);
```

Comment:
As I mentioned to you before, thinking twice, it is better not to mix the import with the install, might be confusing, please create another test module for it.